generators/service-redis/templates/swift/instrumentation.swift

```diff
@@ -3,12 +3,13 @@ import CloudEnvironment
 import SwiftRedis
 
 let redis = Redis()
-var redisCredentials: RedisCredentials!
 
-func initializeServiceRedis() throws {
+func initializeServiceRedis(cloudEnv: CloudEnv) throws -> RedisCredentials {
+    let redisCredentials: RedisCredentials!
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment. You can also get rid of `redisLocal` now there is no module-level var.

Reply:
done